src/button/button.scss

```diff
@@ -11,7 +11,7 @@ $iui-button-padding-large: $iui-xs * 6;
 
 @mixin iui-button {
   --_iui-button-active-stripe-inset: initial;
-  --_iui-button-foreground-color: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-2));
+  --_iui-button-text-color: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-2));
```

Comment:
What's the reason for this change? @veekeys

This color is used for the icon as well.
![image](https://user-images.githubusercontent.com/9084735/166487899-3eb1b042-5441-48d2-865d-e38f1d11a429.png)

Reply:
amm, but not directly. I just went through the variable use, and it was used only for the `color`, so for me it is much clearer when it is named that way.. We can think of other name, just foreground color does not sound obvious what it is, for me at least